packages/services/pkg/schema-type/arrayAbiTypes.go

```diff
@@ -0,0 +1,5 @@
+package schematype
+
+func ArrayAbiTypeToStaticAbiType(schemaType SchemaType) SchemaType {
+	return (schemaType - UINT8_ARRAY)
```

Comment:
should we add a check here that the input type is actually an array type and throw an error else? thinking of cases where users incorrectly input `BYTES`, `STRING` or a static type